torch/distributed/algorithms/ddp_comm_hooks/powerSGD_hook.py

```diff
@@ -264,6 +265,29 @@ def __init__(
         # Turn on if compression / decompression computation is a bottleneck.
         self.batch_tensors_with_same_shape = batch_tensors_with_same_shape
 
+    def __getstate__(self):
+        r"""
+        Returns a ``Dict[str, Any]`` which will be pickled and saved.
+        ``process_group`` is not serializable and excluded from
```

Comment:
Add a logger warning here indicating this?

Reply:
Added logger warning and a check in the unit test.